src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,20 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
the action is not duplicate, the kind of target operation is.

but we can rename it to conflicting_targets which is maybe explaining it better.